src/components/scene/inspector.js

```diff
@@ -0,0 +1,52 @@
+/* global AFRAME */
+var registerComponent = require('../../core/component').registerComponent;
+var pkg = require('../../../package');
+
+var INSPECTOR_URL = pkg.homepage + 'releases/' + pkg.version + '/aframe-inspector.min.js';
+
+module.exports.Component = registerComponent('inspector', {
+  schema: {
+    url: {default: INSPECTOR_URL}
+  },
+
+  init: function () {
+    this.onKeydown = this.onKeydown.bind(this);
+  },
+
+  play: function () {
+    window.addEventListener('keydown', this.onKeydown);
+  },
+
+  pause: function () {
+    window.removeEventListener('keydown', this.onKeydown);
+  },
+
+  onKeydown: function (evt) {
+    // Alt + Ctrl + e
+    var shortcutPressed = evt.keyCode === 69 && evt.ctrlKey && evt.altKey;
+    if (!this.data || !shortcutPressed) { return; }
+    this.injectInspector();
+    this.toggleInspector();
+  },
+
+  toggleInspector: function () {
+    if (AFRAME.inspector.opened) {
```

Comment:
it's possible that this code gets run before the `<script>` is actually injected by `injectInspector`.

even checking for `AFRAME.inspectorInjected` isn't enough. you'll want to add a callback or Promise for the `script.onload` for `injectInspector`.